test/common/modules/live_objects_helper.js

```
@@ -167,5 +195,5 @@ define(['shared_helper'], function (Helper) {
     }
   }
 
-  return (module.exports = new LiveObjectsHelper());
+  return (module.exports = LiveObjectsHelper);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)